modules/msk/calculatenoise/main.nf

```diff
@@ -0,0 +1,79 @@
+process CALCULATENOISE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://ghcr.io/msk-access/sequence_qc:0.2.5' :
```

Comment:
Try changing this URL to: `ghcr.io/msk-access/sequence_qc:0.2.5 . I think the singularity test isn't able to pull the image.